[2025-08-26T09:01:22.132+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.model_evaluation manual__2025-08-26T09:00:21.277583+00:00 [queued]>
[2025-08-26T09:01:22.149+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.model_evaluation manual__2025-08-26T09:00:21.277583+00:00 [queued]>
[2025-08-26T09:01:22.149+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-08-26T09:01:22.171+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): model_evaluation> on 2025-08-26 09:00:21.277583+00:00
[2025-08-26T09:01:22.180+0000] {standard_task_runner.py:60} INFO - Started process 315 to run task
[2025-08-26T09:01:22.186+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'model_evaluation', 'manual__2025-08-26T09:00:21.277583+00:00', '--job-id', '180', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline.py', '--cfg-path', '/tmp/tmp3nxmmvwb']
[2025-08-26T09:01:22.190+0000] {standard_task_runner.py:88} INFO - Job 180: Subtask model_evaluation
[2025-08-26T09:01:22.314+0000] {task_command.py:423} INFO - Running <TaskInstance: ml_pipeline.model_evaluation manual__2025-08-26T09:00:21.277583+00:00 [running]> on host 1729dc9a5378
[2025-08-26T09:01:22.461+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarah' AIRFLOW_CTX_DAG_ID='ml_pipeline' AIRFLOW_CTX_TASK_ID='model_evaluation' AIRFLOW_CTX_EXECUTION_DATE='2025-08-26T09:00:21.277583+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-26T09:00:21.277583+00:00'
[2025-08-26T09:01:33.651+0000] {logging_mixin.py:188} INFO - STDOUT:
[2025-08-26T09:01:33.653+0000] {logging_mixin.py:188} INFO -  üèÉ View run luminous-goose-9 at: https://dagshub.com/SamMebarek/mlops-som.mlflow/#/experiments/0/runs/2cc1d3c3e7aa405aaf3b9d9d02854227
üß™ View experiment at: https://dagshub.com/SamMebarek/mlops-som.mlflow/#/experiments/0
Evaluation completed. R2: 0.9998, MAE: 1.1356
[2025-08-26T09:01:33.654+0000] {logging_mixin.py:188} INFO - STDERR:
[2025-08-26T09:01:33.658+0000] {logging_mixin.py:188} INFO -  2025-08-26 09:01:28,198 - evaluation.utils.common - INFO - YAML charg√© : /opt/***/project/config/config.yaml
2025-08-26 09:01:28,218 - evaluation.utils.common - INFO - YAML charg√© : /opt/***/project/config/params.yaml
2025-08-26 09:01:28,231 - evaluation.utils.common - INFO - R√©pertoire cr√©√© ou existant : /opt/***/project/evaluation
2025-08-26 09:01:30,761 - __main__ - INFO - Loaded processed data: /opt/***/project/data/processed/clean_data.csv (shape=(108060, 14))
2025-08-26 09:01:30,915 - __main__ - INFO - Loaded model: /opt/***/project/models/xgb_model.pkl
2025-08-26 09:01:31,044 - __main__ - INFO - Computed metrics: {'r2_score': 0.9997932180824073, 'mae': 1.135571365098727}
2025-08-26 09:01:31,053 - evaluation.utils.common - INFO - R√©pertoire cr√©√© ou existant : /opt/***/project/evaluation
2025-08-26 09:01:31,073 - evaluation.utils.common - INFO - JSON sauvegard√© : /opt/***/project/evaluation/scores.json
2025-08-26 09:01:31,073 - __main__ - INFO - Metrics saved to: /opt/***/project/evaluation/scores.json
2025/08/26 09:01:31 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet

2025-08-26 09:01:32,213 - __main__ - INFO - Metrics logged to MLflow.
[2025-08-26T09:01:33.660+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-26T09:01:33.698+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=model_evaluation, execution_date=20250826T090021, start_date=20250826T090122, end_date=20250826T090133
[2025-08-26T09:01:33.763+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-26T09:01:33.847+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
